openapi/index.yaml

```diff
@@ -541,6 +541,42 @@ paths:
           description: Cannot regenerate service key.
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/organizations/{organization_fiscal_code}/logo":
+    parameters:
+      - name: organization_fiscal_code
+        in: path
+        type: string
```

Comment:
Do we have a better definition than `string`? Can we use [`OrganizationFiscalCode`](https://github.com/pagopa/io-functions-commons/blob/d615113237d2d65a55a9b6e1f81b762f12f9c108/openapi/definitions.yaml#L287)